src/lib/ng-package/entry-point/entry-point.ts

```diff
@@ -61,45 +61,44 @@ export class NgEntryPoint {
 
   /** Absolute file path of the entry point's source code entry file. */
   public get entryFilePath(): string {
-    return path.resolve(this.basePath, this.entryFile);
+    return resolve(this.basePath, this.entryFile);
   }
 
   /** Whether or not the entrypoint is secondary */
   public get isSecondaryEntryPoint(): boolean {
-    return !!this.secondaryData;
+    return this.secondaryData !== undefined;
```

Comment:
Nit: in general I prefer the previous style.

Reply:
Reverted.